L1Trigger/TrackFindingTracklet/data/MemPrints/CleanTrack/.keep

```diff
@@ -0,0 +1 @@
+
```

Comment:
why do we need these empty directories ?

Reply:
OK removed.